QuantConnect.IBAutomater/IBAutomater.cs

```diff
@@ -259,6 +259,13 @@ public StartResult Start(bool waitForExit)
                             _ibAutomaterInitializeEvent.Set();
                         }
 
+                        // a Java exception was thrown
+                        if (e.Data.StartsWith("Exception"))
+                        {
+                            _lastStartResult = new StartResult(ErrorCode.JavaException, e.Data);
+                            _ibAutomaterInitializeEvent.Set();
+                        }
+
```

Comment:
I'm thinking we could maybe combine these new code additions into a single function